test/cqlpy/test_secondary_index.py

```diff
@@ -192,6 +192,15 @@ def test_create_index_if_not_exists2(cql, test_keyspace, cassandra_bug):
         with pytest.raises(InvalidRequest, match="already exists"):
             cql.execute(f"CREATE INDEX IF NOT EXISTS {index_name} ON {table}(v2)")
 
+# Verify that oversized index names are cleanly rejected  as InvalidRequest
+# Reproduces issue #20755
+def test_create_index_oversized_name(cql, test_keyspace, cassandra_bug):
```

Comment:
I see you marked this a cassandra_bug - what happens on Cassandra? Does it accept the long name? Does it fail in an ugly way?
The Cassandra documentation doesn't mention a length limit on this name, but I wouldn't be surprised if oversized names will also fail on it.

Reply:
I've removed it